src/uct/ib/rc/base/rc_iface.h

```diff
@@ -332,6 +332,7 @@ uct_rc_iface_put_send_op(uct_rc_iface_send_op_t *op)
 {
     uct_rc_iface_t *iface = op->iface;
     ucs_assert(op->flags & UCT_RC_IFACE_SEND_OP_FLAG_IFACE);
+    op->flags = UCT_RC_IFACE_SEND_OP_FLAG_IFACE;
```

Comment:
don't see how it fixes the problem .. need to drop FLAG_INUSE (not FLAG_IFACE) in uct_rc_txqp_purge_outstanding()